src/Microsoft.AspNetCore.SignalR/TypedClientBuilder.cs

```diff
@@ -0,0 +1,243 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Only change other than resources was here https://github.com/aspnet/SignalR/blob/db5d2c5c71116eaa9dc043063bd6e2a86a132594/src/Microsoft.AspNetCore.SignalR/TypedClientBuilder.cs#L63
Changed CreateType method to CreateType info. Other than that its the same.